docs/basic-features/data-fetching.md

```diff
@@ -233,6 +233,9 @@ Instead you can use `process.cwd()` which gives you the directory where Next.js
 ```jsx
 import fs from 'fs'
 import path from 'path'
+import util from 'util'
+
+const readFile = util.promisify(fs.readFile)
```

Comment:
It can potentially use `fs.promises` instead 🤔

Reply:
https://nodejs.org/dist/latest-v10.x/docs/api/fs.html

Reply:
How does that work with the ESNEXT import syntax?

```
import { promises } from 'fs'
```
?

Reply:
Exactly! You might want to do `import { promises as fs } from 'fs'` though.